app/models/govuk_publishing_components/component_fixture.rb

```diff
@@ -0,0 +1,15 @@
+module GovukPublishingComponents
+  ComponentFixture = Struct.new(:id, :data) do
+    def name
+      id.humanize
+    end
+
+    def pretty_data
+      JSON.pretty_generate(data).gsub('\\n', "\n    ").gsub(/"(\w*)":/, '\1:').strip
```

Comment:
Another ported model from here:
https://github.com/alphagov/govuk-component-guide/blob/master/app/models/fixture.rb#L7

It creates a hash that can be put into a `<pre>` block with correct formatting. I agree the naming should be better.